### DISCUSSION TITLE
error when starting it

### DISCUSSION THREAD

**Discussion by suintila** [jef/streetmerchant#1232] [Help] [answered]
I get this message when I start it 
{
  "stack": "Error: Could not find browser revision 818858. Run \"PUPPETEER_PRODUCT=firefox npm install\" or \"PUPPETEER_PRODUCT=firefox yarn install\" to download a supported Firefox browser binary.\n    at ChromeLauncher.launch (C:\\streetmerchant\\node_modules\\puppeteer\\lib\\cjs\\puppeteer\\node\\Launcher.js:79:23)\n    at async PuppeteerExtra.launch (C:\\streetmerchant\\node_modules\\puppeteer-extra\\dist\\index.cjs.js:129:25)\n    at async main (C:\\streetmerchant\\build\\index.js:36:15)\n    at async Timeout.loopMain [as _onTimeout] (C:\\streetmerchant\\build\\index.js:72:9)" 

how can I solve that

**Comment by FehamIsmail** [comment]
same here

**Comment by isma373** [comment]
I get also error but a different one: 

`TSError: ⨯ Unable to compile TypeScript:
src/notification/discord.ts(2,21): error TS2307: Cannot find module 'discord.js' or its corresponding type declarations.

    at createTSError (C:\Users\Tanganica\streetmerchant\node_modules\ts-node\src\index.ts:513:12)
    at reportTSError (C:\Users\Tanganica\streetmerchant\node_modules\ts-node\src\index.ts:517:19)
    at getOutput (C:\Users\Tanganica\streetmerchant\node_modules\ts-node\src\index.ts:752:36)
    at Object.compile (C:\Users\Tanganica\streetmerchant\node_modules\ts-node\src\index.ts:968:32)
    at Module.m._compile (C:\Users\Tanganica\streetmerchant\node_modules\ts-node\src\index.ts:1056:42)
    at Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Object.require.extensions.<computed> [as .ts] (C:\Users\Tanganica\streetmerchant\node_modules\ts-node\src\index.ts:1059:12)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
`

Do I need to download separately discord.js?

**Comment by isma373** [comment, marked as answer]
> I get also error but a different one:
> 
> `TSError: ⨯ Unable to compile TypeScript:
> src/notification/discord.ts(2,21): error TS2307: Cannot find module 'discord.js' or its corresponding type declarations.
> 
> ```
> at createTSError (C:\Users\Tanganica\streetmerchant\node_modules\ts-node\src\index.ts:513:12)
> at reportTSError (C:\Users\Tanganica\streetmerchant\node_modules\ts-node\src\index.ts:517:19)
> at getOutput (C:\Users\Tanganica\streetmerchant\node_modules\ts-node\src\index.ts:752:36)
> at Object.compile (C:\Users\Tanganica\streetmerchant\node_modules\ts-node\src\index.ts:968:32)
> at Module.m._compile (C:\Users\Tanganica\streetmerchant\node_modules\ts-node\src\index.ts:1056:42)
> at Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
> at Object.require.extensions.<computed> [as .ts] (C:\Users\Tanganica\streetmerchant\node_modules\ts-node\src\index.ts:1059:12)
> at Module.load (internal/modules/cjs/loader.js:928:32)
> at Function.Module._load (internal/modules/cjs/loader.js:769:14)
> at Module.require (internal/modules/cjs/loader.js:952:19)
> ```
> 
> `
> 
> Do I need to download separately discord.js?

I did download and it worked: npm install discord.js

**Comment by ghost** [comment]
> I did download and it worked: npm install discord.js

This worked for me too

**Comment by zimm1** [comment]
> I get this message when I start it
> {
> "stack": "Error: Could not find browser revision 818858. Run "PUPPETEER_PRODUCT=firefox npm install" or "PUPPETEER_PRODUCT=firefox yarn install" to download a supported Firefox browser binary.\n at ChromeLauncher.launch (C:\streetmerchant\node_modules\puppeteer\lib\cjs\puppeteer\node\Launcher.js:79:23)\n at async PuppeteerExtra.launch (C:\streetmerchant\node_modules\puppeteer-extra\dist\index.cjs.js:129:25)\n at async main (C:\streetmerchant\build\index.js:36:15)\n at async Timeout.loopMain [as _onTimeout] (C:\streetmerchant\build\index.js:72:9)"
> 
> how can I solve that

Try to delete node_modules folder inside the project folder, and then reinstall with `npm i`
https://stackoverflow.com/questions/62820586/error-could-not-find-browser-revision-756035-run-npm-install
